assets/js/google_map.test.js

```diff
@@ -0,0 +1,250 @@
+const utils = require('./google_map');
+const { google } = require("./__mock__/maps");
+const { test, expect, describe, beforeEach, afterEach, beforeAll, afterAll } = require("@jest/globals");
+const { LatLng, Label } = require('./google_map');
+const origGoogle = global.google;
+
+beforeAll(() => {
+    global.google = google; // mock
+});
+
+afterAll(() => {
+    global.google = origGoogle; // restore
+});
+
+describe("Test middle point", () => {
+    const { middlePoint } = utils;
+    test('Middle of an integer array [1,2,3]', () => {
```

Comment:
Should the function name be `arithmeticMean` or `mean`?

Reply:
Done